jenkins/e2e-image/e2e-runner.sh

```diff
@@ -121,8 +119,10 @@ function fetch_published_version_tars() {
     echo "Using published version $bucket/$build_version (from ${published_version})"
     fetch_tars_from_gcs "gs://${bucket}/${path}" "${build_version}"
     unpack_binaries
-    # Set CLUSTER_API_VERSION for GKE CI
-    export CLUSTER_API_VERSION=$(echo ${build_version} | cut -c 2-)
+    if [[ "${set_cluster_version}" == "true" ]]; then
```

Comment:
Curiously enough, if it's `true` or `false` you can just say `if "${set_cluster_version}"; then` and it will use the programs. Not sure if that's a good or bad thing, though.